src/frontend/app/features/cloud-foundry/tabs/cloud-foundry-summary-tab/cloud-foundry-summary-tab.component.scss

```diff
@@ -1,5 +1,5 @@
 .summary {
   display: flex;
-  height: 100%;
+  min-height: 100%;
```

Comment:
With the current layout - the last card - the recent apps is supposed to size to the remaining space - with this change, this is no longer the case.

Reply:
What are we trying to avoid by having the recent apps shrink to fit the remaining size rather than being it's natural height?

Reply:
The intent was that the remaining space would be used for the apps list - so the more height there was, the more of the list that would be visible, rather than fixing some arbitrary height.

See: https://github.com/cloudfoundry-incubator/stratos/pull/2483 for a fix which I think keeps this behaviour but avoid the overlap issue.